packages/react-adminer/src/components/CellEditInput.tsx

```diff
@@ -37,7 +37,13 @@ const CellEditInput: React.FC<Props> = ({
 		const payload = { [propertyName]: value };
```

Comment:
1. Bylo by hezčí udělat nějakou funkci v utils složce. Něco jako "fixFormat()" a ta to bude vracet.
2. Co když bude initValue "prazda", lepší by bylo to číst z konfigurace